client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -0,0 +1,189 @@
+{{#let @form as |Form|}}
+  <Form.Section @title="Analysis">
+    <Ui::Question as |Q|>
+      <Q.Prompt>
+        Describe the development assumptions. These assumption could be the result of existing conditions, local policies, etc.
+      </Q.Prompt>
+      <AdditionalText
+        @additionalTextTitle="Additional Guidance"
+      >
+        <p>
+          Consider the guidance on soft site criteria provided in the CEQR Technical Manual Chapter, "Establishing the Analysis Framework" (pages 2-6 and 2-7, 2-10 and 2-11), in conjunction with neighborhood-specific trends in the "Study Area."
+        </p>
+        <p>
+          Summarize the criteria used to identify soft sites while including the following information:
+          <ul>
+            <li>Rationale for selecting Projected Development Sites and Potential Development Sites</li>
+            <li>Possible use conversions</li>
+            <li>Expansions or sites with bulk or uses that would be legalized in the future with the Proposed Actions</li>
+            <li>Other Sites that may be excluded from the analysis, if applicable (e.g., rent stabilized    residential buildings, which are difficult to legally demolish due to tenant relocation requirements)  Please note that dwelling unit sizes typically range between 850 – 1,000 gsf.</li>
+          </ul>
+        </p>
```

Comment:
This isn't valid HTML. Lists can't go inside paragraphs. You just need to move the closing `</p>` to be between  "…information:" and the opening `<ul>`.

Reply:
I'm surprised lint didn't catch that?